PennMobile/Courses/Models/CoursesModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CourseModel.swift
+//  PennMobile
+//
+//  Created by Anthony Li on 9/30/22.
+//  Copyright © 2022 PennLabs. All rights reserved.
+//
+
+import Foundation
+
+/// Manages the fetching of courses.
+class CoursesModel: ObservableObject {
```

Comment:
Our code base usually names ObservableObjects as __ViewModel, so to fit the style, maybe renaming it to CoursesViewModel might be ideal